.github/workflows/create-release.yml

```diff
@@ -0,0 +1,29 @@
+# Github Action to create a release with goreleaser
+name: Create Release
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v3
+      -
+        name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v4
+        with:
+          distribution: goreleaser
+          version: latest
```

Comment:
Do we have a process for using goreleaser for non-latest tags?

Reply:
Never mind... I think this `latest` is referring to *goreleaser*, is that right?

Reply:
Yes, correct, just saying use the latest goreleaser binary... which maybe we should change.